transport.go

```diff
@@ -472,15 +488,18 @@ func (t *HTTPSyncTransport) SendEvent(event *Event) {
 	)
 
 	response, err := t.client.Do(request)
-
 	if err != nil {
```

Comment:
I assume that `err` and `response` are mutually exclusive, and only one of them can be `nil` at the time?

Reply:
https://golang.org/pkg/net/http/#Client.Do

>  On error, any Response can be ignored. A non-nil Response with a non-nil error only occurs when CheckRedirect fails [...]